guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -55,7 +55,53 @@ endif::[]
 endif::[]
 ifdef::orcharhino[]
 ifdef::suse_linux_enterprise_server[]
```

Comment:
I wondered if we could somehow set `suse_linux_enterprise_server` instead.

Btw, I recently learned you combine ifdefs with `and` by using `+` (https://docs.asciidoctor.org/asciidoc/latest/directives/ifdef-ifndef/#checking-multiple-attributes) so you can remove a lot of nested `ifdef` statements in this document:
```asciidoc
ifdef::orcharhino+suse_linux_enterprise_server[]
```

Reply:
I would have to test if that works for downstream docs too, but yet. I am currently investigating how to handle Client OS-specific content. I would like to keep it as is for now and then, possibly, fix all of those instances in a follow-up PR.

Reply:
I would like it if we upstream had some sort of preview what's rendered downstream. Right now that really isn't true for orcharhino and that also makes reviews harder.